SPEC/BITSWAP.md

```diff
@@ -4,13 +4,37 @@
 * [bitswap.unwant](#bitswapunwant)
 * [bitswap.stat](#bitswapstat)
 
-#### `bitswap.wantlist`
+#### `bitswap.unwant`
 
-(not spec'ed yet)
+> Removes a given block from your wantlist
 
-#### `bitswap.unwant`
+##### `Go` **WIP**
+
+##### `JavaScript` - ipfs.bitswap.unwant(cid)
+
+`cid` is a [cid][cid] which can be passed as:
+
+- CID, a CID instance
+- String, the base58 encoded version of the multihash
+
+##### `Go` **WIP**
+
+### `bitswap.wantlist`
+
+> Returns the wantlist, optionally limited by peerID
+
+#### `Go` **WIP**
+
+#### `JavaScript` - ipfs.bitswap.wantlist([peerId])
+
+```JavaScript
+ipfs.bitswap.wantlist((err, list) => console.log(list))
+
+//[ { Wantlist object }, ... ]
```

Comment:
Can you elaborate on the shape of the "Wantlist object" here? I think we usually try to in these specs.